lib/suse/connect/api.rb

```diff
@@ -89,6 +90,7 @@ def make_request(method, url, options = {})
         options[:accept_encoding] = 'gzip, deflate'
         options[:headers] = {
           'RMT' => system_uuid.strip,
+          'RMT-HOST' => host_username.to_s.strip,
```

Comment:
Introducing a new header here would need new code in SCC to handle it. 
Currently, SCC is looking for the host_system_username in the uuid field (app/models/registration_proxy.rb:29): 
![image](https://user-images.githubusercontent.com/582520/133788691-8491f919-67ee-4b36-83da-cdc667d13102.png)

Reply:
Sorry, I just saw your explanation in `glue`. That makes sense then.